distribution/ddf-common/src/main/resources/bin/ddf.bat

```diff
@@ -80,6 +43,6 @@ EXIT /B
 :STOP_SOLR
 IF "%SOLR_CLIENT%" == "HttpSolrClient" (
```

Comment:
Looks like this bug was already present, but this solr_client lost its definition in a previous change meaning that we never stop solr on shutdown.

Reply:
Good catch!